quasar/main.tf

```diff
@@ -475,3 +475,28 @@ resource "aws_db_instance" "quasar" {
   performance_insights_enabled    = true
   enabled_cloudwatch_logs_exports = ["postgresql", "upgrade"]
 }
+
+resource "random_string" "admin_password" {
+  # Adding to generate secure password for VPN AD Domain.
+  length = 24
+
+  # We can't use '@' or '$' in MySQL passwords.
+  override_special = "!#%&*()-_=+[]{}<>:?"
```

Comment:
We can remove this override, since this isn't making a MySQL password, right?

Reply:
AD passwords have a funky history of limits and complexity, you know [like this announcement from this year](https://techcommunity.microsoft.com/t5/Azure-Active-Directory-Identity/Removal-of-the-16-character-limit-for-passwords-in-Azure-AD/ba-p/565275) 🙄. I think 24 character length with no special characters makes sense, but can definitely get into the password complexity limits/details if you think it's worth it.

Reply:
Okay, sure. If the goal is to include no special characters, could you change this to `special = false` and remove the comment referencing MySQL?

Reply:
Can do!